app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/LabTestRecordsAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package ca.bc.gov.bchealth.ui.healthrecord.individual
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import ca.bc.gov.bchealth.databinding.ItemHealthRecordsAbstractBinding
+
+/**
+ * @author Pinakin Kansara
+ */
+class LabTestRecordsAdapter(
+    private val labTestRecordClickListener: LabTestRecordClickListener
+) :
+    ListAdapter<HealthRecordItem, LabTestRecordsAdapter.LabTestRecordsViewHolder>(LabTestRecordDiffCallBacks()) {
+
+    fun interface LabTestRecordClickListener {
+        fun onItemClick(record: HealthRecordItem)
+    }
+
+    class LabTestRecordsViewHolder(val binding: ItemHealthRecordsAbstractBinding) :
+        RecyclerView.ViewHolder(binding.root)
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): LabTestRecordsViewHolder {
+        val view = ItemHealthRecordsAbstractBinding.inflate(
+            LayoutInflater.from(parent.context), parent, false
+        )
+
+        return LabTestRecordsViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: LabTestRecordsViewHolder, position: Int) {
+        val record = getItem(position)
+        holder.binding.tvVaccineName.text = record.title
+        holder.binding.tvVaccineStatus.text = record.date
```

Comment:
Please rename tvVaccineStatus to tvDesc and add number of tests as shown in the design